protofire/omen-exchange
# Fix/#1532

-how to test 
-go to import section of rinkeby add this link https://reality.eth.link/app/#!/question/0x33d847ecfe8422ab920abb4615a606a933207241a067c45a3c613fc70832cd10
-should load good
